drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/PhantomJSDriverFactory.java

```diff
@@ -35,17 +33,14 @@
  * PhantomJSDriver.
  *
  * @author <a href="kpiwko@redhat.com">Karel Piwko</a>
- *
+ * @author <a href="mjobanek@redhat.com">Matous Jobanek</a>
```

Comment:
Also it gets outdated pretty soon and... hints to code ownership (by individual)